website/pages/playground.tsx

```diff
@@ -5,7 +5,10 @@ import { Playground } from '../components/Playground';
 
 export default function PlaygroundPage() {
   return (
-    <Layout>
+    <Layout
+      title="Check how it works | uniforms"
+      description="Check how uniforms work, and enjoy the schema-first approach of one of the most popular React form libraries and enjoy building forms like never before."
```

Comment:
This sentence sounds really strange `and enjoy` phrase is repeated twice and it's only once delimited by a comma. If we want it to stay I think it should look like this:

```suggestion
      description="Check how uniforms work, enjoy the schema-first approach of one of the most popular React form libraries, and enjoy building forms like never before."
```

But honestly, I think this should be rephrased.

Reply:
After consultation: "Check how uniforms work, enjoy the schema-first approach of one of the most popular React form libraries, and build forms as easily as never before."